Triphouse-project/style.css

```diff
@@ -50,4 +79,54 @@ height: 10px;
     padding-bottom: 122px;
     font-size: 25px;
     font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
-  }
\ No newline at end of file
+  }
+
+  footer {
+    background: #10367C;
+    width: 1440px;
+    margin: 0px auto;
+    padding:10px 0;
+  }
+
+  .footer-content {
+    display: flex;
+	  Justify-content:space-around;
+    flex-direction: row
+  }
+  .footer-content__list {
+    margin: 120px 0px;
+    padding: 20px;
+    
+  }
+
+  .footer-content__list ul:first-child {
+    margin-bottom: 20px;
+  }
+
+  ul {
+    list-style: none;
+    padding: 20px;
+  }
+  li {
+    height: 50px;
+    font-family: "Roboto";
+    font-size: medium;
+    color: #ffff;
+  }
+
+  li:before {
```

Comment:
Use margin for `li` item instead `before`.

Reply:
fixed